### PR TITLE
Update philips.js

### PR DESCRIPTION
**Pull request by adaenzer**
added 3261331P6
this is the P6 model of the available Ambiance Still P7 model

### CODE REVIEW

**Comment by sjorge** [comment]
Can you try getting the colorTemp range for this device? 

https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

**Comment by adaenzer** [comment]
here's the output:
Zigbee2MQTT:debug 2021-06-04 12:47:13: Received MQTT message on 'zigbee2mqtt/light_bedroom/set' with data '{"read": {"cluster": "lightingColorCtrl", "attributes": ["colorTempPhysicalMin", "colorTempPhysicalMax"]}}'
Zigbee2MQTT:debug 2021-06-04 12:47:13: Publishing 'set' 'read' to 'light_bedroom'
Zigbee2MQTT:debug 2021-06-04 12:47:13: Received Zigbee message from 'light_bedroom', type 'readResponse', cluster 'lightingColorCtrl', data '{"colorTempPhysicalMax":454,"colorTempPhysicalMin":153}' from endpoint 11 with groupID 0
Zigbee2MQTT:info  2021-06-04 12:47:13: MQTT publish: topic 'zigbee2mqtt/light_bedroom', payload '{"color_mode":"xy","linkquality":115,"state":"ON"}'
Zigbee2MQTT:info  2021-06-04 12:47:13: Read result of 'lightingColorCtrl': {"colorTempPhysicalMin":153,"colorTempPhysicalMax":454}

it interacts exactly the same as the other hue lamps i have. it also has the same color temp limits.

File: devices/philips.js [hunk 1]
```diff
@@ -933,6 +933,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['3261331P6'],
+        model: '3261331P6',
+        vendor: 'Philips',
+        description: 'Hue white ambiance Still',
+        meta: {turnsOffAtBrightness1: true},
+        extend: hueExtend.light_onoff_brightness_colortemp(),
```
Comment (sjorge) [inline devices/philips.js:942]: Can you update this to
```
extend: hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 454]}),
```

Those are the min/max from your read results

**Comment by sjorge** [comment]
> it interacts exactly the same as the other hue lamps i have. it also has the same color temp limits.

We only started adding them recently, most hue bulbs were added long ago and we have no data available so they default to 150-500. We try to get it for new  entries, of course if you have some of those missing the entries it would be good to complete them too.

**Comment by adaenzer** [comment]
so  i'll close this and open a new pull request with the added info for the other lamps i have, too.

**Comment by sjorge** [comment]
> so i'll close this and open a new pull request with the added info for the other lamps i have, too.

If you’re willing to update some of the others you have access to that would be great

**Pull request closed without merging**